lib/redmine/info.rb

```diff
@@ -31,8 +31,12 @@ def web_server
       def session_store
         if Rails.application.config.session_store.name == "ActionDispatch::Session::RedisStore"
           options = Rails.application.config.session_options[:servers].first
-          version = Redis.new(options).info['redis_version']
-          "Redis #{version}"
+          cache_box_info = Redis.new(options).info
+          if cache_box_info['server_name'].eql?('valkey')
+            "Valkey #{cache_box_info['valkey_version']}"
+          else
+            "Redis #{cache_box_info['redis_version']}"
+          end
```

Comment:
```suggestion
          server_info = Redis.new(options).info
          if server_info['server_name'].eql?('valkey')
            "Valkey #{server_info['valkey_version']}"
          else
            "Redis #{server_info['redis_version']}"
          end
```

IMO better fitting name (instead of our docker service name)